README.md

```diff
@@ -0,0 +1,43 @@
+This file describes the work done and steps to perform test on the xacro package.
```

Comment:
I'd much rather see this README describe what Xacro does.  There could be another section (further down) that describes the differences between the ROS 1 and ROS 2 port, but it's honestly not all that important.